src/renderer/vt/VtSequences.cpp

```diff
@@ -345,3 +345,91 @@ using namespace Microsoft::Console::Render;
 {
     return _Write("\x1b[24m");
 }
+
+// Method Description:
+// - Writes a sequence to tell the terminal to start italicizing text
+// Arguments:
+// - <none>
+// Return Value:
+// - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
+[[nodiscard]] HRESULT VtEngine::_BeginItalics() noexcept
+{
+    return _Write("\x1b[3m");

```

Comment:
I wonder if there's an optimization we can take later that'll let us compress SGRs together when we know we're writing a bunch. `\x1b[40m`+`\x1b[31m`+`\x1b[4m`+`\x1b[3m` => `\x1b[40;31;4;3m`?

Reply:
(moved to #3016)